src/services/todos.js

```diff
@@ -0,0 +1,26 @@
+import { client, checkError } from './client';
+
+export async function fetchToDos() {
+  const response = await client.from('todos').select();
```

Comment:
we could add a `.order()` here too so that the todos always come back in order on refresh